build/task-build.js

```diff
@@ -68,12 +69,13 @@ async function babelDir(source, target) {
 
     if (sourceFile.endsWith('.js')) {
       const { code } = await transpile(sourceFile, { sourceMaps: true });
-      await fs.writeFile(targetFile, code);
+      promises.push(fs.writeFile(targetFile, code));
     } else {
-      const buffer = await fs.readFile(sourceFile);
-      await fs.writeFile(targetFile, buffer);
+      promises.push(fs.copy(sourceFile, targetFile));
```

Comment:
Nope, the files can be processed in whatever order, might as well try to do it in parallel where possible